module/VuFind/src/VuFind/Db/Service/PluginManager.php

```diff
@@ -46,6 +46,7 @@ class PluginManager extends \VuFind\ServiceManager\AbstractPluginManager
      * @var array
      */
     protected $aliases = [
+        'changeTracker' => ChangeTrackerService::class,
```

Comment:
We should probably be consistent in capitalization of aliases -- I notice that changeTracker and userResource are mixed-case while everything else is all lowercase. I think it's probably best to consistently use all-lowercase aliases. Even though this is slightly less readable, it makes case-insensitive service resolution less complicated, since we can rely on the aliases set here being consistently-cased.